src/server/kiwix-serve.cpp

```diff
@@ -63,7 +63,7 @@ void usage()
             << "\t-h, --help\t\tPrint this help" << std::endl << std::endl
             << "\t-a, --attachToProcess\tExit if given process id is not running anymore" << std::endl
             << "\t-d, --daemon\t\tDetach the HTTP server daemon from the main process" << std::endl
-            << "\t-i, --address\t\tListen only on this ip address, all available ones otherwise" << std::endl
+            << "\t-i, --address\t\tListen only on this ip address, all available ones otherwise, or specify special values to accept connections from ipv4/ipv6/all devices" << std::endl
```

Comment:
Change to

Listen only on the specified IP address. Specify 'ipv4', 'ipv6' or 'all' to listen on all IPv4, IPv6 or both types of addresses, respectively (default: all).